GoogleForCreators/web-stories-wp
# Add snapping for dragging elements from library

Depends on #5337, do not merge first.

## Summary
Adds snapping for shapes, media, text sets.


## Relevant Technical Choices



## To-do



## User-facing changes
- Snapping lines now appear when dragging elements from library.


## Testing Instructions
1. Add a rectangular shape to the page.
2. Now start dragging a media from the library to the page.
3. Verify the dragged media display snapping lines against the page, the design space, and also the shape that's on the page.
4. Verify the same steps for Shapes and Text Sets.


---



Fixes #1979 
Fixes #5230
